src/com/dotcms/rest/api/v1/system/AppConfigurationHelper.java

```diff
@@ -56,10 +97,36 @@ public Object getMenuData(final HttpServletRequest request) {
 	 * @return The list of required system properties.
 	 */
 	public Object getConfigurationData(final HttpServletRequest request) {
-		final ConfigurationResource configurationResource = new ConfigurationResource();
 		final Response configurationResponse = configurationResource.list(request);
```

Comment:
Same thing here: Instead of going through the resource class, use the Helper class for the ConfigurationResource.